src/main/java/com/grafana/opentelemetry/OpenTelemetryConfig.java

```diff
@@ -45,13 +45,34 @@ public MeterRegistry openTelemetryMeterRegistry(OpenTelemetry openTelemetry, Clo
 
     @Bean
     public OpenTelemetry openTelemetry(Optional<AutoConfiguredOpenTelemetrySdk> sdk) {
-        return sdk.<OpenTelemetry>map(AutoConfiguredOpenTelemetrySdk::getOpenTelemetrySdk)
-                       .orElse(OpenTelemetry.noop());
+        OpenTelemetry openTelemetry = sdk.<OpenTelemetry>map(AutoConfiguredOpenTelemetrySdk::getOpenTelemetrySdk)
+                .orElse(OpenTelemetry.noop());
+        addLogAppender(openTelemetry);
+        return openTelemetry;
+    }
+
+    private void addLogAppender(OpenTelemetry openTelemetry) {
+        //the openTelemetry object is not used yet, but it will be used in the future, when the global otel instance is not used by default anymore
+
+        try {
+            Class.forName("ch.qos.logback.classic.Logger");
```

Comment:
Rather than having the nested try-catch here, could we move the Class check to the addLogger methods in each class and change the method so it returns a boolean.  True, if the logger was added, and false otherwise.

Reply:
done :smile: